app/javascript/packs/one-time-code-input.js

```diff
@@ -1,6 +1,19 @@
 import OneTimeCodeInput from '@18f/identity-one-time-code-input';
 
-const input = document.querySelector('.one-time-code-input');
-if (input) {
-  new OneTimeCodeInput(/** @type {HTMLInputElement} */ (input)).bind();
+const fakeField = /** @type {HTMLInputElement?} */ (document.querySelector('.one-time-code-input'));
+
+if (fakeField) {
+  const el = `<input type="hidden" name="code" id="code" class="hidden-input">`;
+  fakeField.insertAdjacentHTML('afterend', el);
+  fakeField.removeAttribute('name');
+  fakeField.removeAttribute('id');
```

Comment:
Do we need to remove the ID? Based on the `aria-label` below, could we leave it for the label to be associated correctly?

Reply:
I thought one of the main issues was that the name and id were associated with the cached values on web browsers. If we don't remove them, we would have both inputs have id of code. Unless we want to not have id be code for the hidden input .

Reply:
We could test to make sure, but I'd sorta expect the problem to stem from the `name` association, but not ID.

Reply:
You are right, so long as ID is unique so I just didn't include ID for the hidden value.